<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AlcoCalc</class>
 <widget class="QDialog" name="AlcoCalc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>278</width>
    <height>286</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Alcohol Calculator</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>owned percentage</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="ownedBox"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>
final amount of 
solution</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="AmountSolEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>desired percentage</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="desireBox"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>calculate</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
